### ISSUE TITLE
fix(parametric_action_distribution): sum kl divergence over event_ndims in parametric action distribution

### ISSUE THREAD

**Issue by clement-bonnet** [instadeepai/jumanji#141]
### Is your feature request related to a problem? Please describe
The KL divergence is not summed over the event dim for multi-dimensional action distribution. This makes `kl_divergence` return a batch of marginal KLs instead of one joint KL.

### Describe the solution you'd like
In the `kl_divergence` method, we could add the following post-processing:
```python
        if self._event_ndims == 1:
            kl = jnp.sum(kl, axis=-1)
        else:
            assert self._event_ndims == 0
```

### Remark
MATHS: We need to make sure the joint KL is the sum of marginal KL.
